pyproject.toml

```diff
@@ -30,15 +30,15 @@ classifiers = [
   "Typing :: Typed",
 ]
 dynamic = ["version"]
-dependencies = ["pybamm", "cookiecutter"]
+dependencies = ["pybamm", "copier", "jinja2-time"]
```

Comment:
Yes, that seems fair enough to me, I am curious what others have to say on this.

Reply:
If I understand correctly, pybamm is only a dependency for the template, right? So it should not be needed as a dependecy on the project.

Reply:
I think in order for the entry points to work we would need `pybamm` as a dependency inside the template as the `BaseModel` for all the models is derived from `pybamm`.

Reply:
In that case, we can drop the entry points from the project when the template is ready, so that `pybamm-cookiecutter` can provide just the proposed CLI.